### PR TITLE
docs: eased up GridForm recommendation in FormInputs

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Changes `FormInputs` to recommend `GridForm` only for _groups_ of the inputs.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/W6mHtayHlCVuvUWsr4MDcq/Reporting-and-Analytics?node-id=49%3A749
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Review by CharlieBliss — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.8-alpha.fad6e5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fce67bffd8cdd1c3118a495--gamut-preview.netlify.app>https://5fce67bffd8cdd1c3118a495--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fce67bffd8cdd1c3118a495>Deploy Logs</a></p>

File: packages/styleguide/stories/Core/Atoms/FormInputs/FormInputs.stories.mdx [hunk 1]
```diff
@@ -26,8 +26,8 @@ import {
 
 Input atoms tied 1:1 with values to be submitted in a form.
 
-You should generally never use these directly as a Gamut consumer;
-instead, use `GridForm` to auto-generate these for you.
+You should generally never several of these grouped together directly as a Gamut consumer;
```
Comment (jakemhiller) [inline packages/styleguide/stories/Core/Atoms/FormInputs/FormInputs.stories.mdx:29]: is this what was intended?

```suggestion
You should generally never use several of these grouped together directly as a Gamut consumer;
```

Comment (JoshuaKGoldberg) [reply packages/styleguide/stories/Core/Atoms/FormInputs/FormInputs.stories.mdx]: eek

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.0.8-alpha.085323.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fce717ced8c760114d6f142--gamut-preview.netlify.app>https://5fce717ced8c760114d6f142--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fce717ced8c760114d6f142>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@21.1.6-alpha.43d7bd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd13d51e218d8219ad37e5d--gamut-preview.netlify.app>https://5fd13d51e218d8219ad37e5d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd13d51e218d8219ad37e5d>Deploy Logs</a></p>

**Pull request merged**